examples/sample-app/application-template-stibuild.json

```diff
@@ -110,7 +110,8 @@
           "type": "STI",
           "stiStrategy": {
             "image": "openshift/ruby-20-centos",
-            "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
+            "from": { "name": "ruby-20-centos" },
+	    "scripts": "https://raw.githubusercontent.com/openshift/ruby-20-centos/master/.sti/bin"
```

Comment:
i think this change is correct.  we'll want e2e to be testing using the repo ref.  (we need to test both, of course, but we can't put everything in e2e)